benefits/in_person/urls.py

```diff
@@ -10,7 +10,9 @@
     path(
         "eligibility/", admin.site.admin_view(views.EligibilityView.as_view()), name=routes.name(routes.IN_PERSON_ELIGIBILITY)
     ),
-    path("token/", admin.site.admin_view(views.token), name=routes.name(routes.IN_PERSON_ENROLLMENT_TOKEN)),
+    path(
+        "token/", admin.site.admin_view(views.InPersonTokenView.as_view()), name=routes.name(routes.IN_PERSON_ENROLLMENT_TOKEN)
```

Comment:
This endpoint is only relevant for `littlepay` enrollment, e.g. should be `/in_person/enrollment/littlepay/token`